### PR TITLE
fix: trim address when inputting new asset

### LINKED ISSUE #3847: Trim user address when adding new assets in asset management
## Problem Definition

Ensure that there is no whitespace surrounding the token address when saving a new token.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #3847

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.
- [ ] Trim address when inputting new asset

### CODE REVIEW

File: frontend/app/src/components/asset-manager/AssetForm.vue [hunk 1]
```diff
@@ -525,6 +532,10 @@ export default class AssetForm extends Vue {
     }
     return identifier;
   }
+
+  private sanitizeAddress(address: string) {
+    return address.replace(/\s/g, '');
+  }
```
Comment (yabirgb) [inline frontend/app/src/components/asset-manager/AssetForm.vue:538]: Isn't it better to use trim here? https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/String/Trim

Comment (lukicenturi) [reply frontend/app/src/components/asset-manager/AssetForm.vue:538]: Yes trim will do it. But I think, isn't address also cannot have `space` too in the middle, why dont remove it altogether? @yabirgb

**Comment by lukicenturi** [comment]
the test seems to failed, need this to be merged https://github.com/rotki/rotki/pull/3849

**Review by kelsos — APPROVED** [review]
lgtm, we just need to rebase I guess

File: docs/changelog.rst [hunk 2]
```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :fix`3847` Remove whitespace when inputting address in new asset form.
```
Comment (kelsos) [inline docs/changelog.rst:5]: This should probably be `:bug:`

**Pull request merged**